controllers/reconcile.go

```diff
@@ -684,12 +678,10 @@ func (r *FrontendReconciliation) getFrontendPaths() []string {
 	frontendPaths := r.Frontend.Spec.Frontend.Paths
 	defaultPath := fmt.Sprintf("/apps/%s", r.Frontend.Name)
 	defaultBetaPath := fmt.Sprintf("/beta/apps/%s", r.Frontend.Name)
```

Comment:
Beta path can go too?

Reply:
Thought about it, I added the removal as a separate commit now, thanks!